cis/Issues/official/description.xml

```diff
@@ -0,0 +1,33 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Stichting Centraal Informatie Systeem</en>
+		<nl>Demo Stichting Centraal Informatie Systeem</nl>
+	</Name>
+	<ShortName>
+		<en>Demo CIS</en>
+		<nl>Demo CIS</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>official</CredentialID>
+	<Description>
+		<en>Digitaal Paspoort Verzekeren</en>
+		<nl>Digitaal Paspoort Verzekeren</nl>
+	</Description>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.cis.official.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.cis.official.html</nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="InternationaleSanctieCheck">
```

Comment:
The most issuers use lowerCamelCase for their identifier and they use English identifier names. For consistency, that could be something to consider.

Reply:
changed format to lowerCamelCase